k8s/install/terraform/terraform.tfvars

```diff
@@ -0,0 +1,7 @@
+ssh_key_file = "<ssh_public_key_file_path>"
```

Comment:
I think we can use a template file.
or we can set the default to ssh key to `$HOME/.ssh/id_rsa`

Reply:
added a template file and moved tfvars to git ignore, I don't think defaulting to id_rsa is a good idea, though.